### ISSUE TITLE
Use 1-dim feature vector extracted from pre-trained model to train a new CNN 

### ISSUE THREAD

**Issue by Kelvin-Zhong** [BVLC/caffe#836]
I have saw some other similar issues, but none of them can help me solve this problem. 
I have used python to put all the feature extracted from the pre-train model in the leveldb, and use it as the source to train a new network.

I extracted the output vector of fc6 of imagenet network, which is a 1-dim 4096 vector.
Each value of my leveldb is a datum.SerializedtoString().
and datum.channel = 1, datum.height = 4096, datum.width = 1, and I have set the datum.label.

However, when I use this leveldb as input to train network, the loss function doesn't decrease, so as the Test score #1, and the Test #0 is stuck...

What is the issue? Can anyone provide a simple example on how to use this 1-dim vector to create a right leveldb to train a new network???

Many thanks~~
